PRO58/Src/state_favourite_screen_ui.cpp

```diff
@@ -0,0 +1,218 @@
+#include "state_favourite_screen.h"
```

Comment:
This is more or less a copy paste of search_ui. Maybe refactoring to use common base

Reply:
common base done ;)